swagger/fs_wrapper.go

```diff
@@ -0,0 +1,95 @@
+package swagger
+
+import (
+	"bytes"
+	"html/template"
+	"io"
+	"io/fs"
```

Comment:
We'll need to drop the `go1.15.x` case in the tests matrix here: https://github.com/wI2L/fizz/blob/master/.github/workflows/ci.yml#L19